### PR TITLE
add-osds: followup on 3632b26

### PR DESCRIPTION
**Pull request by guits**
fix a typo in vagrant_variables that cause a networking issue for
containerized scenario.

Signed-off-by: Guillaume Abrioux <gabrioux@redhat.com>

### CODE REVIEW

**Comment by guits** [comment]
jenkins test luminous-add_osds

**Comment by guits** [comment]
jenkins test luminous-add_osds_container

**Comment by guits** [comment]
jenkins test luminous-add_osds

**Comment by guits** [comment]
jenkins test luminous-add_osds_container

**Comment by guits** [comment]
jenkins test luminous-add_osds_container

File: tests/functional/centos/7/add-osds-container/group_vars/all [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ---
 ceph_origin: repository
 ceph_repository: community
+containerized_deployment: true
```
Comment (leseb) [inline tests/functional/centos/7/add-osds-container/group_vars/all]: you forgot `docker: true`

**Comment by guits** [comment]
jenkins test luminous-add_osds_container

**Comment by guits** [comment]
jenkins test playbook syntax

**Comment by alfredodeza** [comment]
This PR broke every ceph-volume test that uses the `simple` sub-command because we rely on osd tests running on OSD hosts because we depend on group_vars being set explicitly for OSD nodes, like number of devices.

This is beyond frustrating as it happens **continuously**. Can you please enable some sort of verification for ceph-volume tests?

**Pull request merged**